examples/flags/flags.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"github.com/akamensky/argparse"
+	"os"
+)
+
+func main() {
+	// Create new parser object
+	parser := argparse.NewParser("flags", "Simple example of argparse flags")
+
+	// Create verbose flag
+	verb := parser.Flag("v", "verbose", &argparse.Options{Help: "Enable verbose mode"})
+
+	// Parse input
+	err := parser.Parse(os.Args)
+	if err != nil {
+		// In case of error print error and print usage
+		// This can also be done by passing -h or --help flags
+		fmt.Print(parser.Usage(err))
```

Comment:
I think it would also be good to exit in case of error so that application won't proceed running the code after catching this error :)

Reply:
Will fix. I suppose I was still under the impression of using python's equivalent.